addons/website/views/res_config_settings_views.xml

```diff
@@ -20,23 +20,33 @@
         <field name="arch" type="xml">
             <xpath expr="//form" position="inside">
                 <app data-string="Website" string="Website" name="website" groups="website.group_website_designer">
-                    <setting type="header" string="Settings of Website">
+                    <setting
+                        type="header"
+                        string="Settings of Website"
+                        documentation="/applications/websites/website/configuration/multi_website.html"
+                    >
                         <field name="website_id" options="{'no_open': True, 'no_create': True}" title="Settings on this page will apply to this website"/>
                         <button name="action_website_create_new" type="object" string="+ New Website" class="btn btn-link"/>
                     </setting>
                     <block title="Website Info" id="website_info_settings">
                         <setting>
                             <div class="content-group">
                                 <div class="row mt8">
-                                    <label class="col-lg-3" string="Domain" for="website_domain"/>
+                                    <div class="col-lg-3">
+                                        <label string="Domain" for="website_domain"/>
+                                        <a href="https://www.odoo.com/documentation/17.0/applications/websites/website/configuration/domain_names.html#domain-name-website-map" title="Documentation" class="o_doc_link" target="_blank"></a>
+                                    </div>
                                     <field name="website_domain" placeholder="https://www.odoo.com" title="Display this website when users visit this domain"/>
                                 </div>
                                 <div class="row mt8">
                                     <label class="col-lg-3" string="Homepage URL" for="website_homepage_url"/>
                                     <field name="website_homepage_url" placeholder="/" title="Main page of your website served to visitors"/>
                                 </div>
                                 <div class="row mt8">
-                                    <label class="col-lg-3" string="Languages" for="language_ids"/>
+                                    <div class="col-lg-3">
+                                        <label string="Languages" for="language_ids"/>
+                                        <a href="https://www.odoo.com/documentation/17.0/applications/websites/website/configuration/translate.html" title="Documentation" class="o_doc_link" target="_blank"></a>
+                                    </div>
```

Comment:
@rdeodoo Since the div for the text is so small, the ? button goes to the next line. I checked with other languages and other settings and it's Odoo's default behavior, it's just that it doesn't look good in this specific case.

![image](https://github.com/odoo/odoo/assets/46810843/102c22c9-c976-46cd-b557-6ba29048e0b5)